feat(changelog): postgresql-and-mysql-added-managed-database-support-ipam 2023-05-12 (#1645)
scaleway/docs-content: changelog/may2023/2023-05-12-postgresql-and-mysql-added-managed-database-support-ipam.mdx

---

> title: Managed Database support IPAM

Reviewer: ```suggestion
title: Managed Databases now support IPAM
```

status: added
author:

>     fullname: 'Join the #[https://scaleway-community.slack.com/archives/CJBLQUR5J] channel on Slack.'

Reviewer: ```suggestion
    fullname: 'Join the #database channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2023-05-12
category: databases
product: postgresql-and-mysql
---

> Managed DBs support IPAM integration.

Reviewer: ```suggestion
Managed Databases now support IPAM integration.
```
